ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/DC1-L2LEAF1B.md

```diff
@@ -462,17 +462,23 @@ ip route vrf MGMT 0.0.0.0/0 192.168.200.5
 
 ### IP IGMP Snooping Summary
 
-IGMP snooping is globally enabled.
+| Global Settings | Values |
+| --------------- | ------ |
+| IGMP snooping | Enabled |
+| Fast-leave | True |
+| Proxy | False |
 
+#### IP IGMP Snooping Vlan 120 Settings Summary
 
-| VLAN | IGMP Snooping |
-| --- | --------------- |
-| 120 | disabled |
```

Comment:
There should not be a change in this documentation output, likely an issue in documentation template.

Reply:
There is no issue. I refactored the previous documentation. I added the vlan 120 to the heading (line 471) and then listed it's settings. I did this refactor to print more settings on VLAN basis.

Reply:
Perhaps I should have added another table in the global settings (above line 471) to depict which vlans are enabled/disabled.

Reply:
I can make the changes if the documentation style doesn't look good.